generate_logs.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+set -eu
+
+generate_log_file() {
+  local date=$1
+  local sleep_seconds=$2
+
+  echo "## Generate logs for ${date}"
+  rm -rf logs/access-${date}.log.gz
+
+  echo "Run dummer"
+  DUMMER_DATE="${date}" bundle exec dummer -c dummer.conf -w 4 -d
+
+  echo "Waif for ${sleep_seconds} seconds"
+  sleep ${sleep_seconds}
+  bundle exec dummer stop
+
+  echo "Generated $(cat logs/access.log | wc -l) lines"
+
+  echo "Sort logs..."
+  cat logs/access.log | sort > logs/access-${date}.log
```

Comment:
It doesn't matter, but `cat` is unneeded in this case. 
`sort logs/access.log > /logs/access-${date}.log` is enough.